components/ui/EditorButtons.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  toggleEmphasisCommand,
+  toggleStrongCommand,
+  wrapInBulletListCommand,
+  wrapInOrderedListCommand,
+  wrapInBlockquoteCommand,
+  toggleInlineCodeCommand,
+  wrapInHeadingCommand,
+} from "@milkdown/preset-commonmark";
+import {
+  AiOutlineItalic,
+  AiOutlineOrderedList,
+  AiOutlineUnorderedList,
+  AiOutlineStrikethrough,
+  AiOutlineUnderline,
+  AiOutlineBold,
+} from "react-icons/ai";
+import { toggleStrikethroughCommand } from "@milkdown/preset-gfm";
+import { VscQuote, VscCode } from "react-icons/vsc";
+import { CgFormatHeading } from "react-icons/cg";
+import { Editor, CmdKey } from "@milkdown/core";
+import { callCommand } from "@milkdown/utils";
+import { useEditor } from "@milkdown/react";
+import "@milkdown/theme-nord/style.css";
+import React from "react";
+
+const EditorButtons = () => {
+  const editor = useEditor(() => Editor.make());
+
+  function call<T>(command: CmdKey<T>, payload?: T) {
+    return editor.get()?.action(callCommand(command, payload));
+  }
+  return (
+    <div className="flex w-full gap-4 pb-4">
+      <button onClick={() => call(wrapInHeadingCommand.key)}>
+        <CgFormatHeading size={20} />
```

Comment:
We should replace these buttons with custom HeroIcons that we add in components/icons.
Also we should uninstall react-icons.

https://heroicons.com/